floor/test/integration/view_test/view_test.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:async';
+
+import 'package:floor/floor.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:path/path.dart' hide equals;
+import 'package:sqflite/sqflite.dart' as sqflite;
+import 'package:sqflite_ffi_test/sqflite_ffi_test.dart';
+
+import '../dao/dog_dao.dart';
+import '../dao/name_dao.dart';
+import '../dao/person_dao.dart';
+import '../model/dog.dart';
+import '../model/name.dart';
+import '../model/person.dart';
+
+part 'view_test.g.dart';
+
+@Database(version: 2, entities: [Person, Dog], views: [Name])
```

Comment:
💡We can get of all the migration related code in this file as we're not testing migrations.

```suggestion
@Database(version: 1, entities: [Person, Dog], views: [Name])
```